cypress/integration/home.spec.ts

```diff
@@ -2,3 +2,48 @@ it('should redirect to home page', () => {
   cy.visit('/');
   cy.location('pathname').should('eq', '/');
 });
+
+it('should create a new task', () => {
+  cy.get('[data-testid="add-task-input"]').type('Created by Cypress');
+  cy.get('[data-testid="add-task-btn"]').click();
+  cy.get('[data-testid="task-item"]').its('length').should('eq', 1);
+  cy.get('[data-testid="task-label"').contains('Created by Cypress');
```

Comment:
I suggest removing this line since the assert was made previously.

Reply:
Resolved on 72d9849a9848b1dbeda5bd2b520b5dae66786573